docs/reference/streamfield/blocks.md

```diff
@@ -412,7 +418,7 @@ All block definitions accept the following optional keyword arguments:
            ('person', blocks.StructBlock([
                ('first_name', blocks.CharBlock()),
                ('surname', blocks.CharBlock()),
-               ('photo', ImageChooserBlock(required=False)),
+               ('photo', ImageBlock(required=False)),
```

Comment:
I can see that the 'is decorative' checkbox is probably redundant in a situation like this, but surely the added value of the contextual alt text field makes it worthwhile?

Realistically I think there'll always be a significant portion of Wagtail devs who aren't particularly switched on to accessibility considerations, and won't be actively thinking "is `ImageBlock` or `ImageChooserBlock` the right choice here?" when putting their models together. For those devs, our goal should be to plant the idea that `ImageBlock` is the right one to reach for by default, even if they're not actively aware of the reasoning _why_ it's the right one. (Indeed, that's a large part of the reason I originally proposed `ImageBlock` as the name rather than, say, `ImageWithAltChooserBlock`...) For that reason, I think there's a good case to make for the documentation erring on the side of using `ImageBlock` more frequently to reinforce that idea, even if the reality is more nuanced.

Reply:
@Stormheg @SaptakS What are your opinions on this?
I think having alt text is a good trade-off to using the `Image_block` despite the redundancy of the `decorative` checkbox.

Reply:
I tend to agree with @gasman on this . I think most people might  go for the default option shown in one example, and it's important to tell them that in most cases `ImageBlock` might be the right one. The developers still definitely have the choice to use `ImageChooserBlock` if they want. I agree @thibaudcolas that in this case it's definitely a developer's decision, but I feel more frequently we talk about `ImageBlock` in the examples, the more widely it will get adapted (and also hopefully developers and editors will learn a little more about the different scenarios and cases where they might want it to be decorative or not)